src/main/scala/nl.knaw.dans.easy.bag2deposit/DepositPropertiesFactory.scala

```diff
@@ -29,45 +29,47 @@ case class DepositPropertiesFactory(configuration: Configuration, idType: IdType
   def create(bagInfo: BagInfo, ddm: Elem): Try[PropertiesConfiguration] = Try {
     val ddmIds: NodeSeq = ddm \ "dcmiMetadata" \ "identifier"
 
-    def getBaseUrn(versionOf: UUID) = configuration.bagIndex.getURN(versionOf).unsafeGetOrThrow
+    def getBaseUrn(versionOf: UUID) = {
+      bagSource match {
+        case VAULT => configuration.bagIndex.getURN(versionOf).unsafeGetOrThrow
+        case FEDORA => bagInfo.baseUrn.getOrElse(throw InvalidBagException("version-of without base-urn"))
```

Comment:
i'm a bit lost here. Does this mean we expect a base-urn to be present in the bag-info? is `easy-fedora-to-bag` putting it there?

Reply:
It is not yet implemented in `easy-fedora-to-bag`, but we discussed it that way. Otherwise we would need access to fedora to get it. It is removed by `easy-convert-bag-to-deposit` so it doesn't hurt validation further down the chain.